addons/react-linked-input/react-linked-input.js

```diff
@@ -8,184 +68,234 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
-'use strict';
-
-var emptyFunction = require('fbjs/lib/emptyFunction');
-var invariant = require('fbjs/lib/invariant');
-var warning = require('fbjs/lib/warning');
-
-var hasReadOnlyValue = {
-  'button': true,
-  'checkbox': true,
-  'image': true,
-  'hidden': true,
-  'radio': true,
-  'reset': true,
-  'submit': true,
-};
-
-function _assertSingleLink(inputProps) {
-  invariant(
-    inputProps.checkedLink == null || inputProps.valueLink == null,
-    'Cannot provide a checkedLink and a valueLink. If you want to use ' +
-    'checkedLink, you probably don\'t want to use valueLink and vice versa.'
-  );
-}
-function _assertValueLink(inputProps) {
-  _assertSingleLink(inputProps);
-  invariant(
-    inputProps.value == null && inputProps.onChange == null,
-    'Cannot provide a valueLink and a value or onChange event. If you want ' +
-    'to use value or onChange, you probably don\'t want to use valueLink.'
-  );
-}
-
-function _assertCheckedLink(inputProps) {
-  _assertSingleLink(inputProps);
-  invariant(
-    inputProps.checked == null && inputProps.onChange == null,
-    'Cannot provide a checkedLink and a checked property or onChange event. ' +
-    'If you want to use checked or onChange, you probably don\'t want to ' +
-    'use checkedLink'
-  );
-}
-
-var loggedTypeFailures = {};
-function getDeclarationErrorAddendum(owner) {
-  if (owner) {
-    var name = owner.getName();
-    if (name) {
-      return ' Check the render method of `' + name + '`.';
-    }
-  }
-  return '';
-}
-
-/**
+          "use strict";
+          var emptyFunction = require("fbjs/lib/emptyFunction");
+          var invariant = require("fbjs/lib/invariant");
+          var warning = require("fbjs/lib/warning");
+
+          var hasReadOnlyValue = {
+            button: true,
```

Comment:
I did run it through prettier - was wary of trying to take specific parts of the changes because I might miss a closing bracket somewhere.